ru/stage3.json

```
@@ -41,10 +41,22 @@
   },
   "proposal-accessible-object-hasownproperty": {
     "title": "Accessible Object.prototype.hasOwnProperty()",
-    "description": "Proposal for an Object.hasOwn() method to make Object.prototype.hasOwnProperty() more accessible."
+    "description": "Предложение по добавлению метода Object.hasOwn(), который делает Object.prototype.hasOwnProperty() более доступным."
   },
   "proposal-resizablearraybuffer": {
     "title": "In-Place Resizable and Growable ArrayBuffers",
     "description": null
+  },
+  "proposal-array-find-from-last": {
+    "title": "Array.prototype.findLast and Array.prototype.findLastIndex",
+    "description": "Методы по поиску соответствия с конца масси для элементов и их индексов."
```

```suggestion
    "description": "Методы по поиску соответствия с конца массива для элементов и их индексов."
```
Just a small drive-by typo fixup

---

thanks!